app/views/sitemap/index.xml.haml

```diff
@@ -0,0 +1,17 @@
+!!! XML
+%urlset{xmlns: "http://www.sitemaps.org/schemas/sitemap/0.9"}
+  - for page in @pages
+    %url
+      %loc= root_url + page
```

Comment:
It'd be nice to build URLs using the helpers rather than crafting them like this, since if anything changes to these URLs then this will break without us knowing. Perhaps `@pages` could contain `shop_url`, `map_url`, etc.